src/Middleware/RequestThrottling/src/RequestThrottlingExtensions.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Aspnetcore.RequestThrottling;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public static class RequestThrottlingExtensions
```

Comment:
Doc comments for all public classes and members, even if they say obvious things. You should be seeing warnings/errors about these missing.

Reply:
I believe most of the other middleware have example messages you could borrow from.